Sources/Client/WebSocket/WebSocket.swift

```diff
@@ -360,6 +352,23 @@ extension WebSocket: WebSocketProviderDelegate {
         
         return nil
     }
+    
+    private func handleTypingEvent(_ event: Event) {
+        guard case .typingStart(let user, _, _) = event else {
+            return
+        }
+        
+        if let timerControl = typingEventTimeoutTimerControls[user] {
+            timerControl.cancel()
+            typingEventTimeoutTimerControls[user] = nil
+        }
+        
+        if case .typingStart = event {
```

Comment:
Why this? `event` is `typingStart`, you already check it at L357

Reply:
Fixed